gradle.properties

```diff
@@ -22,11 +22,5 @@ geodeVersion = 1.11.0-SNAPSHOT
 geodeRepositoryUrl =
 geodeReleaseUrl =
 
-# dependency versions
-assertjVersion = 3.6.2
-awaitilityVersion = 1.7.0
-junitVersion = 4.12
-mockitocoreVersion = 2.19.1
-log4jVersion = 2.11.0
-systemrulesVersion = 1.16.1
-micrometerVersion = 1.2.0
+# dependency versions not found in geode-all-bom
+micrometerVersion = 1.2.1
```

Comment:
amused that micrometer isn't in the bom.  wonder if it should be 🤔

looks good otherwise though!

Reply:
micrometer-core is, but not registry (used as a client-side test)

Reply:
ah that makes sense, forgot about that!